core/modules/filter/js/filter.js

```diff
@@ -140,12 +141,19 @@ Backdrop.behaviors.editorImageDialog = {
       });
     });
 
+    // Initialise styles of Dialog.
+    // Hide the left-hand (library) part of the dialog form.
+    $(".image-form-left").css({"display":"none"});
+    $(".editor-dialog").removeClass("editor-dialog-with-library");
+    // Set the class for the left-hand part.
+    $(".image-form-right").addClass("image-form-right-full");
+
     $newToggles.click(function(e) {
       var $link = $(e.target);
       if ($link.is('.editor-image-toggle') === false) {
         return;
       }
-
+      // find the first ancestor of link
```

Comment:
Make this sentence case and with a period.

Reply:
Done.